conf/uppmax-devel.config

```diff
@@ -23,24 +23,24 @@ params {
 
 process {
   queue = 'devel'
-  container = "${params.containerPath}/rnafusion-dev.simg"
+  container = "${params.containerPath}/rnafusion_dev.img"
```

Comment:
Why using this extension?
I believe that the latest one is SIF, the previous was SIMG, and the first was IMG.

Reply:
It was already mentioned here once: https://gitter.im/nextflow-io/nextflow?at=5bf2e24b5f5c925040818e66